### PR TITLE
Skip get previous tx when cellbase

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/services/sync/get-blocks.ts [hunk 1]
```diff
@@ -46,7 +46,12 @@ export default class GetBlocks {
         const checkTx = new CheckTx(tx, this.url)
         const addresses = await checkTx.check(lockHashes)
         if (addresses.length > 0) {
-          for (const input of tx.inputs!) {
+          const inputs = tx.inputs!
+          for (let i = 0; i < inputs.length; ++i) {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/sync/get-blocks.ts]: The loop can start from 1.

File: packages/neuron-wallet/src/services/sync/get-blocks.ts [hunk 2]
```diff
@@ -46,7 +46,12 @@ export default class GetBlocks {
...
+            if (i === 0) {
+              continue
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/sync/get-blocks.ts]: For what reason the first input should be skipped?

Comment (Keith-CY) [reply packages/neuron-wallet/src/services/sync/get-blocks.ts]: It's a cellbase and has no previous output.

File: packages/neuron-wallet/src/services/sync/get-blocks.ts [hunk 3]
```diff
@@ -46,7 +46,12 @@ export default class GetBlocks {
         const checkTx = new CheckTx(tx, this.url)
         const addresses = await checkTx.check(lockHashes)
         if (addresses.length > 0) {
-          for (const input of tx.inputs!) {
+          const inputs = tx.inputs!
+          for (let i = 0; i < inputs.length; ++i) {
+            if (i === 0) {
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/sync/get-blocks.ts]: That's the first tx of a block, not the first input of a tx.

Comment (classicalliu) [reply packages/neuron-wallet/src/services/sync/get-blocks.ts]: fixed

**Pull request merged**